man/mergelist.Rd

```diff
@@ -0,0 +1,172 @@
+\name{mergelist}
+\alias{mergelist}
+\title{Merge multiple data.tables}
+\description{
+  Faster merge of multiple \code{data.table}s.
+}
+\usage{
+  mergelist(l, on, cols,
+    how = c("left","inner","full","right","semi","anti","cross"),
+    mult, copy = TRUE,
+    join.many = getOption("datatable.join.many"))
+}
+\arguments{
+  \item{l}{ \code{list} of \code{data.table}s to merge. }
+  \item{on}{ \code{character} vector of column names to merge on, shared across all inputs; when missing, the \emph{key} of \emph{join-to} table is used. }
```

Comment:
i'm a bit confused by the wording:

> key of join-to table is used

Reply:
"shared across all inputs" could be removed, because it feels like it is not possible to customize that for each join